src/Components/TopContent.tsx

```diff
@@ -1,117 +1,162 @@
-import React from 'react'
-import { Menubar } from 'primereact/menubar';
-import { MenuItem } from 'primereact/menuitem';
-import { useNavigate } from 'react-router-dom';
-import { Divider } from 'primereact/divider';
-import { InputText } from 'primereact/inputtext';
-import { Button } from 'primereact/button';
-import { Card } from 'primereact/card';
-export default function TopContent() {
-
-    const navigate = useNavigate()
+import React, { useState } from "react";
+import { Menubar } from "primereact/menubar";
+import { MenuItem } from "primereact/menuitem";
+import { useNavigate } from "react-router-dom";
+import { ConfirmDialog, confirmDialog } from "primereact/confirmdialog";
+import { Divider } from "primereact/divider";
+import { InputText } from "primereact/inputtext";
+import { Button } from "primereact/button";
+import { Card } from "primereact/card";
+import { ListBox, ListBoxChangeEvent } from "primereact/listbox";
+import { Producte} from "../data";
+
+interface Libelle {
+  name: string;
+  code: string;
+}
 
-    const items: MenuItem[] = [
+interface VisibilityProps {
+  selectedProduct?: Producte;
+}
+const libelle: Libelle[] = [
+  { name: "Acompte/facture", code: "ACC" },
```

Comment:
we can create two enum, one for name values and another for code value to easy reuse that.